plugins/modules/source_control/gitlab/gitlab_project_members.py

```diff
@@ -186,97 +273,212 @@ def main():
     argument_spec.update(dict(
         api_token=dict(type='str', required=True, no_log=True),
         project=dict(type='str', required=True),
-        gitlab_user=dict(type='str', required=True),
-        state=dict(type='str', default='present', choices=['present', 'absent']),
-        access_level=dict(type='str', required=False, choices=['guest', 'reporter', 'developer', 'maintainer'])
+        gitlab_user=dict(type='list', elements='str'),
+        state=dict(type='str', default='present',
+                   choices=['present', 'absent']),
+        access_level=dict(type='str', required=False, choices=[
+                          'guest', 'reporter', 'developer', 'maintainer']),
+        purge_users=dict(type='list', elements='str', choices=[
+                         'guest', 'reporter', 'developer', 'maintainer']),
+        gitlab_users_access=dict(
+            type='list',
+            elements='dict',
+            options=dict(
+                name=dict(type='str', required=True),
+                access_level=dict(type='str', choices=[
+                                  'guest', 'reporter', 'developer', 'maintainer', 'owner'], required=True),
```

Comment:
It looks like `owner` isn't available elsewhere in this module:
```suggestion
                                  'guest', 'reporter', 'developer', 'maintainer'], required=True),
```